TFT/src/User/Menu/PrintingMenu.c

```diff
@@ -457,12 +461,9 @@ static inline void reDrawProgress(uint8_t prevProgress)
 {
   uint8_t nextProgress = getPrintProgress();
 
-  if (nextProgress != prevProgress)
```

Comment:
Testing my PRs I saw some cases when progress was reduced. So it's better to revert also that change.

Attached one example:

[Ataturk_0.2mm_52m_PLA_Microzey_Siyah_btt.zip](https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/files/8135184/Ataturk_0.2mm_52m_PLA_Microzey_Siyah_btt.zip)

Reply:
Thanks for the example file, it yielded a problem for which the solution you proposed is not addressing its fundamental, rather circumvents it.
The problem is not that the progress has to regress when the first gcode is executed. In my view it shouldn't increase when the gcode file reading advances by reading non-gcode info. 
Hence I fixed the core of the issue.